### PR TITLE
Help cmd

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

**Comment by joshblum** [comment]
depends on #244

**Comment by joshblum** [comment]
screen shot 2016-05-02 at 16 50 55

File: orchestra/bots/staffbot.py [hunk 1]
```diff
@@ -33,6 +33,24 @@ def __init__(self, **kwargs):
         token = settings.SLACK_STAFFBOT_TOKEN
         super().__init__(token, **kwargs)
 
+    def help(self):
+        return self.format_slack_message(
+            'Use `/staffbot` to staff or restaff a Task within Orchestra.',
+            attachments=[
+                {
+                    'pretext': '`/staff <task-id>`',
```
Comment (kkamalov) [inline orchestra/bots/staffbot.py]: should it be `/staffbot staff <task-id>` instead of `/staff <task-id>` ?

File: orchestra/bots/staffbot.py [hunk 2]
```diff
@@ -33,6 +33,24 @@ def __init__(self, **kwargs):
...
+                    'text': ('Ask qualified experts to work on'
+                             ' the given `<task-id>`.'),
+                    'mrkdwn_in': ['text', 'pretext'],
+                },
+                {
+                    'pretext': '`/restaff <task-id> <username>`',
```
Comment (kkamalov) [inline orchestra/bots/staffbot.py]: similar to 41

File: orchestra/bots/staffbot.py (hunk 1 above)
Comment (joshblum) [inline orchestra/bots/staffbot.py]: done

File: orchestra/bots/staffbot.py (hunk 2 above)
Comment (joshblum) [inline orchestra/bots/staffbot.py]: done

**Pull request merged**